libnopegl/src/pipeline_compat.h

```diff
@@ -41,6 +41,7 @@ int ngli_pipeline_compat_update_uniform(struct pipeline_compat *s, int32_t index
 int ngli_pipeline_compat_update_texture(struct pipeline_compat *s, int32_t index, const struct texture *texture);
 void ngli_pipeline_compat_update_texture_info(struct pipeline_compat *s, const struct pgcraft_texture_info *info);
 int ngli_pipeline_compat_update_buffer(struct pipeline_compat *s, int32_t index, const struct buffer *buffer, int offset, int size);
+int ngli_pipeline_compat_update_dynamic_offsets(struct pipeline_compat *s, uint32_t *offsets, size_t nb_offsets);
```

Comment:
can we have `const uint32_t *offsets`?

Reply:
Done.

Reply:
You did it for the pipeline but not the compat layer

Reply:
Done.